scripts/check_style.sh

```diff
@@ -1,5 +1,11 @@
 #!/usr/bin/env bash
 
+. scripts/report_errors.sh
+
+trap report_error_to_github EXIT
```

Comment:
Since we source the file - would it work to move this trap command inside `report_errors.sh`?